utils/group.json

```diff
@@ -0,0 +1,41 @@
+{
+  "SyntaxNodes": [
+    "SyntaxData.swift",
+    "TokenKind.swift",
+    "Syntax.swift",
+    "Trivia.swift",
+    "RawSyntax.swift",
+    "SyntaxKind.swift",
+    "SyntaxNodes.swift",
+    "SyntaxCollections.swift",
+    "SourcePresence.swift",
+    "SyntaxChildren.swift",
+  ],
+  "Utils": [
+    "SyntaxBuilders.swift",
+    "SyntaxFactory.swift",
+    "SyntaxRewriter.swift",
+    "SyntaxClassifier.swift",
+  ],
+  "Diagnostic": [
+    "DiagnosticConsumer.swift",
+    "JSONDiagnosticConsumer.swift",
+    "PrintingDiagnosticConsumer.swift",
+    "DiagnosticEngine.swift",
+    "Diagnostic.swift",
+    "SyntaxVerifier.swift",
+  ],
+  "Position": [
+    "AbsolutePosition.swift",
+    "Utils.swift",
+    "SourceLocation.swift",
+    "SourceLength.swift",
+  ],
+  "Parse": [
+    "SyntaxParser.swift",
+    "IncrementalParseTransition.swift",
+  ],
+  "Internal": [
+    "AtomicCounter.swift",
```

Comment:
Is it required that all files are listed ? "AtomicCounter.swift" doesn't contain any public API.

Reply:
It is required unfortunately. But in practice no users would jump to this group by clicking on any APIs.